packages/squiggle-lang/src/rescript/Distributions/DistributionTypes.res

```diff
@@ -176,11 +177,40 @@ module Constructors = {
     let fromSamples = (xs): t => FromSamples(xs)
     let truncate = (dist, left, right): t => FromDist(ToDist(Truncate(left, right)), dist)
     let inspect = (dist): t => FromDist(ToDist(Inspect), dist)
-    let klDivergence = (dist1, dist2): t => FromDist(ToScore(KLDivergence(dist2)), dist1)
-    let logScoreWithPointResolution = (~prediction, ~answer, ~prior): t => FromDist(
-      ToScore(LogScore(answer, prior)),
-      prediction,
-    )
+    module LogScore = {
+      let distEstimateDistAnswer = (estimate, answer): t => FromDist(
+        ToScore(LogScore(Score_Dist(answer), None)),
+        estimate,
+      )
+      let distEstimateDistAnswerWithPrior = (estimate, answer, prior): t => FromDist(
+        ToScore(LogScore(Score_Dist(answer), Some(prior))),
+        estimate,
+      )
+      let distEstimateScalarAnswer = (estimate, answer): t => FromDist(
+        ToScore(LogScore(Score_Scalar(answer), None)),
+        estimate,
+      )
+      let distEstimateScalarAnswerWithPrior = (estimate, answer, prior): t => FromDist(
+        ToScore(LogScore(Score_Scalar(answer), Some(prior))),
+        estimate,
+      )
+      let scalarEstimateDistAnswer = (estimate, answer): t => FromFloat(
```

Comment:
shouldn't these lines with ``scalarEstimate`` be removed?

Reply:
https://github.com/quantified-uncertainty/squiggle/pull/589/commits/78893320527eac715286a5a209352490d0c1397f